packages/@dcl/ecs/src/components/definitions/OnPointerUp.proto

```diff
@@ -0,0 +1,12 @@
+﻿syntax = "proto3";
+
+import "common/id.proto";
+option (ecs_component_id) = 1061;
+
+message PBOnPointerUp {
+  string identifier = 2;
```

Comment:
Wouldn't be better to use a number? The impact of numeric entity ids was huge

Reply:
done!